src/py21cmfast/inputs.py

```diff
@@ -813,20 +813,17 @@ class AstroParams(StructWithDefaults):
     A_VCB, BETA_VCB: float, optional
         Impact of the DM-baryon relative velocities on Mturn for minihaloes. Default is 1.0 and 1.8, and agrees between different sims. See Sec 2 of Muñoz+21 (2110.13919).
     fR: float, optional
-        Radio efficiency for galaxies, normalised to 1 for modern day galaxies,
-        Set fR<0.1 (can be negative) to turn off radio emission
+        Radio efficiency for galaxies, normalised to 1 for modern day galaxies. Given in log10 units.
     aR: float, optional
         Power-law energy spectra index
     fR_mini: float, optional
-        Radio efficiency for molecularly cooling galaxies (MCG) in mini-halos, normalised to 1 for modern day galaxies,
-        Requires USE_MINI_HALOS=T, set fR_mini<0.1 (can be negative) to turn off radio emission
+        Radio efficiency for molecularly cooling galaxies (MCG) in mini-halos, normalised to 1 for modern day galaxies. Given in log10 units.
     aR_mini: float, optional
         Power-law energy spectra index for mini-halos
-    log10_mbh: float, optional
-        log10 of PBH birth mass in msun
-    log10_fbh: float, optional
-        log10 of PBH fraction, i.e. rho_PBH/rho_dm
-        Allowed range: [-Inf, 0]. Set to below -100 to turn off PBH (both radio and Hawking), log10_fbh > 0.01 triggers error
+    mbh: float, optional
+        PBH birth mass in msun. Given in log10 units.
+    fbh: float, optional
+        PBH fraction, i.e. rho_PBH/rho_dm, allowed range: [0, 1]. Given in log10 units.
```

Comment:
I don't mind it being given in log10 units, but I think it's much cleaner if the name includes the log10, so I prefer the original